acid/acid.py

```diff
@@ -335,8 +338,16 @@ def fallback_view(self, view_name, context=None):               # pylint: disabl
             rendered_children=(fragment.content for fragment in rendered_children),
             local_resource_url=self.runtime.local_resource_url(self, 'public/test_data.json'),
         ))
-        frag.add_frag_resources(acid_fragment)
-        frag.add_frags_resources(rendered_children)
+        try:
+            frag.add_fragment_resources(acid_fragment)
+
+            for rendered_child in rendered_children:
+                frag.add_fragment_resources(rendered_child)
+
+        except:
```

Comment:
Also add a comment here.  It should be clear which is the old way of doing things and what's the new way so that it's easier to clean this up in the future.